extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -258,6 +258,9 @@ export class SubscriptionService {
         this.delegateAuthToViewer_();
         this.startAuthorizationFlow_(false);
         return;
+      } else if (this.platformConfig_['alwaysGrant']) {
```

Comment:
Feel free to remove `else` and instead add some comments about what it does here.

Reply:
I'll add the comment but wont make it independent `if` as someone might just re-order without considering that if `viewerAuthentication` check is kinda pre-requisite to this check.